src/model/Question.ts

```diff
@@ -0,0 +1,16 @@
+export default class Question {
+    prompt: string;
+    context: string;
+    answerKey: string;
+
+    constructor(prompt: string, context: string, answerKey: string) {
+        this.prompt = prompt;
+        this.context = context;
+        this.answerKey = answerKey;
+    }
+}
+
+// const QUESTIONS: Array<Question> = [
```

Comment:
Try not to include comments with snippets of old/unused code, they tend to hang around and get confusing as project grow and team members change.